### PR TITLE
feat: NPG-5359 Add new dreps argument for snapshot tool

### PR DESCRIPTION
**Pull request by Mr-Leshiy**
# Description

Adding a new argument `dreps` for snapshot tool. This argument will provide a missed dreps information for properrly generation of the final snapshot. 
`dreps` is a path to the json file with the following format:
```
{
  "reps": ["0x00588e8e1d18cba576a4d35758069fe94e53f638b6faf7c07b8abd2bc5c5cdee", "0xa6a3c0447aeb9cc54cf6422ba32b294e5e1c3ef6d782f2acff4a70694c4d1663"]
}
```

Fixes # (issue) _if applicable_

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by kukkok3** [comment]
could you update the docs as well? thanks!

File: src/catalyst-toolbox/catalyst-toolbox/src/bin/cli/snapshot/mod.rs [hunk 1]
```diff
@@ -47,13 +50,14 @@ pub struct SnapshotCmd {
 impl SnapshotCmd {
     pub fn exec(self) -> Result<(), Report> {
         let raw_snapshot: RawSnapshot = serde_json::from_reader(File::open(&self.snapshot)?)?;
+        let dreps: Dreps = serde_json::from_reader(File::open(&self.dreps)?)?;
```
Comment (stevenj) [inline src/catalyst-toolbox/catalyst-toolbox/src/bin/cli/snapshot/mod.rs:53]: the dreps file should be optional.  If its not set, it should default to an equivalent of:
```
{
  "dreps": []
}
```

**Review by stevenj — CHANGES_REQUESTED** [review]

File: src/catalyst-toolbox/catalyst-toolbox/src/bin/cli/snapshot/mod.rs (hunk 1 above)
Comment (Mr-Leshiy) [reply src/catalyst-toolbox/catalyst-toolbox/src/bin/cli/snapshot/mod.rs]: fixed

**Review by FelipeRosa — APPROVED** [review]
LGTM

**Pull request merged**